internal/bundle/bundle.go

```diff
@@ -0,0 +1,42 @@
+package bundle
+
+import (
+	"path/filepath"
+
+	"github.com/docker/model-distribution/types"
+)
+
+// Bundle represents a runtime bundle containing a model and runtime config
+type Bundle struct {
```

Comment:
nit: Maybe `GGUFBundle` since it would not fit the shape of other model formats. And in the future we'll likely end up with `SafeTensorsBundle`, etc.